[Fundamentals] Added concepts + slight restructuring (#4076)
cloudflare/cloudflare-docs: content/fundamentals/get-started/concepts/what-is-cloudflare.md

---
pcx-content-type: concept
title: What is Cloudflare?
weight: 1
---

# What is Cloudflare?

> Cloudflare is one of the world’s largest networks of [edge servers](https://www.cloudflare.com/learning/cdn/glossary/edge-server/).

Reviewer: I have reread this like 3 times but IDK if network is supposed to go in there grammatically or networks works too.

Author: I think it's networks, `one of .... network` sounds weird since it's implying there's a plural?

When you add your application to Cloudflare, we use this edge network to sit in between requests and your [origin server](https://www.cloudflare.com/learning/cdn/glossary/origin-server/).

![Cloudflare sits in between requests and your origin server.](/fundamentals/static/images/get-started/website-with-cloudflare.svg)

This [position](/fundamentals/get-started/concepts/how-cloudflare-works/) allows us to do several things — speeding up content delivery and user experience ([CDN](https://www.cloudflare.com/learning/cdn/what-is-a-cdn/)), protecting your website from malicious activity ([DDoS](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/), [Firewall](https://www.cloudflare.com/learning/ddos/glossary/web-application-firewall-waf/)), routing traffic ([Load balancing](/load-balancing/), [Waiting room](/waiting-room/)), and more.

{{<Aside type="note">}}

> For more details about what Cloudflare is and does, visit our [Learning Center](https://www.cloudflare.com/learning/what-is-cloudflare/).

Reviewer: ```suggestion
For more details about what Cloudflare is and does, refer to the [Learning Center](https://www.cloudflare.com/learning/what-is-cloudflare/).
```
